.github/workflows/tests.yml

```
@@ -50,7 +50,7 @@ jobs:
           CI: true
       - name: Send codecov.io stats
         if: matrix.node-version == '14.x'
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
```

```suggestion
        uses: codecov/codecov-action@v4
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
```

needs to be added after the secret has been added (and the name may need to be modified, depending on what it was named)